### DISCUSSION TITLE
Is there a way to hide the right panel ?

### DISCUSSION THREAD

**Discussion by Overclock303** [datalust/seq-tickets#1303] [Help]
Hi, is there a way to hide the right panel : Signals, queries...

I would like to have a full Events display.

**Comment by nblumhardt** [comment]
Hi! You can click the green vertical bar that separates this from the main events view, to hide it temporarily - does this help?

**Reply by Overclock303** [reply]
haha yes !  i m sorry. it was too easy to find.
Thx anyway

Seq is a great tool

**Reply by nblumhardt** [reply]
Awesome, thanks @Overclock303 :-)
